sdk/resources/mgmt/src/test/java/com/azure/management/resources/DeploymentsTests.java

```diff
@@ -116,7 +116,7 @@ public void canPostDeploymentWhatIfOnResourceGroup() throws Exception {
                 .whatIf();
 
         Assertions.assertEquals("Succeeded", result.status());
-        // FIXME: regression?
+        // TODO(not known): regression?
         // Assertions.assertEquals(3, result.changes().size());
```

Comment:
Could you try in PLAYBACK whether these are now OK?

Reply:
Done, also resolved another TODO